cl/opinion_page/templates/opinions.html

```diff
@@ -121,7 +121,7 @@ <h3> <span>Jump To</span> </h3>
           </div>
         {% endif %}
 
-        {% if cluster.sub_opinions.all.first.extracted_by_ocr or "U" in cluster.source and tab == "opinions" %}
+        {% if tab == "opinions" and cluster.sub_opinions.all.first.extracted_by_ocr or "U" in cluster.source %}
```

Comment:
We can improve this check to avoid an extra DB query.  Right now, even with `sub_opinions` prefetched, calling `.all.first` still issues a separate query for the first object.  Switching to `.all.0` will use the prefetched results directly:

```suggestion
        {% if tab == "opinions" and cluster.sub_opinions.all.0.extracted_by_ocr or "U" in cluster.source %}
```